llama_stack/apis/common/errors.py

```diff
@@ -19,3 +24,11 @@ class ModelNotFoundError(ValueError):
     def __init__(self, model_name: str) -> None:
         message = f"Model '{model_name}' not found. Use client.models.list() to list available models."
         super().__init__(message)
+
+
+class DatasetNotFoundError(ValueError):
+    """raised when Llama Stack cannot find a referenced dataset"""
+
+    def __init__(self, dataset_name: str) -> None:
+        message = f"Dataset '{dataset_name}' not found. Use client.datasets() to list available datasets."
```

Comment:
is client.datasets() or client.datasets.list()

Reply:
it is not. see this https://llama-stack.readthedocs.io/en/latest/references/python_sdk_reference/index.html#datasets

Reply:
ah apologies, thanks for the link - fixed